### DISCUSSION TITLE
Low Quality Mode Error

### DISCUSSION THREAD

**Discussion by mbhforum** [tesla-android/issue-tracker#19] [Q&A] [answered]
I have a 2019 Model S running 2021.44-13555 and I am getting the error in my browser when I try to bring up the Android interface (http://3.3.3.1):  **Low Quality Mode Forced by Tesla Android in motion, reload the page while in Park**

There is also a broken image on the page.  Of course I ensured I was in Park and hit refresh in the browser, but still not working.  I have also rebooted the MCU.  This is not a Tesla specific issue as it happens when I connect my MacOS to the Tesla Android Pi as well. 

![image](https://user-images.githubusercontent.com/11696289/168390026-9ff91807-3cd9-4cd7-bdfc-fdff2a193832.jpg)

**Comment by mbhforum** [comment, marked as answer]
I fixed this issue myself.  It turns out my HDMI to MicroHDMI wasnt installed properly.  I followed this tutorial to fix:  https://www.youtube.com/channel/UC1wYvQvRbTHM7R0TjiQSEAg

**Reply by cukal** [reply]
Had the same issue, once installed properly the green LED lights up.
